e2e/logoRedirect.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { test, expect } from '@playwright/test'
+import { IssueData, RoadmapApiResponseSuccess } from '../lib/types';
+import * as fixture from '../tests/fixtures/issueData/ipfs-roadmap-102.json'
+
+test('clicking the logo should navigate users to the homepage', async ({ page, context }) => {
+
+  await context.route('**/api/roadmap**', (route) => route.fulfill({
+    status: 200,
+    body: JSON.stringify({
+      errors: [],
+      data: fixture as unknown as IssueData,
+      pendingChildren: [],
+    } as RoadmapApiResponseSuccess),
+  }));
+
+  await page.goto('http://localhost:3000/roadmap/github.com/ipfs/roadmap/issues/102');
+
+  await page.waitForFunction(() => document.querySelectorAll('.chakra-spinner').length === 0, { timeout: 1000 });
+  expect(await page.locator('.chakra-spinner')).toHaveCount(0);
```

Comment:
playwright says `expect(await page.locator('.chakra-spinner')).toHaveCount(0);` should wait for all spinners to go away, but that wasn't true and I couldn't figure out how to set a longer timeout.

Reply:
it does! However, interestingly, it's about 4sec slower. first is with 

```typescript
await page.locator('.chakra-spinner').waitFor({ state: 'hidden', timeout: 1000 });
```

second is with 

```typescript
await page.waitForFunction(() => document.querySelectorAll('.chakra-spinner').length === 0, { timeout: 1000 });
expect(await page.locator('.chakra-spinner')).toHaveCount(0);
```

third is with 

```typescript
await page.waitForFunction(() => document.querySelectorAll('.chakra-spinner').length === 0, { timeout: 1000 });
```

```bash
╰─ ✘ INT ❯ hyperfine 'yarn test:e2e'                                                                          11.33   28.1G   24%   100%  ─╯
Benchmark 1: yarn test:e2e
  Time (mean ± σ):      7.704 s ±  1.384 s    [User: 2.524 s, System: 0.541 s]
  Range (min … max):    5.195 s …  9.389 s    10 runs
  
╰─ ✔ ❯ hyperfine 'yarn test:e2e'                                                                              10.76   28.2G   24%   100%  ─╯
Benchmark 1: yarn test:e2e
  Time (mean ± σ):      6.029 s ±  1.508 s    [User: 2.171 s, System: 0.430 s]
  Range (min … max):    4.796 s …  9.626 s    10 runs
  
╰─ ✔ ❯ hyperfine 'yarn test:e2e'                                                                              10.30   28.2G   24%   100%  ─╯
Benchmark 1: yarn test:e2e
  Time (mean ± σ):      5.569 s ±  1.395 s    [User: 2.178 s, System: 0.422 s]
  Range (min … max):    4.764 s …  9.478 s    10 runs
```

still, that's the idiomatic playwright way so i'll update it.